src/main/java/SwitchNeonRedBluePage.java

```diff
@@ -0,0 +1,16 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class SwitchNeonRedBluePage {
+
+    WebDriver driver;
+    public SwitchNeonRedBluePage(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public boolean isNintendoSwitchDisplayed(String expectedText){
```

Comment:
the name isDisplayed does not correspond with what is done in the function, in the function you are doing a string.contains